inst/include/interface/rcpp/rcpp_objects/rcpp_growth.hpp

```diff
@@ -44,6 +47,9 @@ class EWAAGrowthInterface : public GrowthInterfaceBase {
  public:
   std::vector<double> weights; /**< weights for each age class */
   std::vector<double> ages;    /**< ages for each age class */
+  std::map<double, double> ewaa; /**< map of ewaa values */
+
+  bool initialized = false;
```

Comment:
Does "initialized" always start with false and used inside of the evaluate method? Could you add a brief documentation for "initialized"?